resources/views/partials/menu-top.blade.php

```diff
@@ -5,13 +5,17 @@
 <div class="menu-top">
     <div class="menu-top-container menu-top-bg">
         <div class="row">
-            <div class="small-12 columns">
+            <div class="small-12 columns position-relative">
                 <div class="vertical-centering title-area">
                     <h1>
-                        <a href="/">
-                            @if(config('app.sub_title') !== null)<span>{{ config('app.sub_title') }}</span>@endif
-                            {{ $site['title'] }}
-                        </a>
+                        <div>
+                            <a href="/">
+                                @if($site['id'] !== 1669)
```

Comment:
Was this area meant to be changed? 

The `@if(config('app.sub_title') !== null)` check should still work if there is going to be any change to this area.

Reply:
No it wasn't I will fix thanks!!